docs/partner_editable/additional_info.adoc

```diff
@@ -1,21 +1,21 @@
 == Test the deployment
 
-When you enroll a new AWS Control Tower–managed account, the deployment sets up the IAM role for {partner-product-short-name}. You can then see the new account show up in the New Relic UI.
+When you enroll a new AWS Control Tower–managed account, the deployment sets up the IAM role and Metrics Streaming Resources for {partner-product-short-name}. You can then see the new account show up in the New Relic UI.
```

Comment:
Looking at the catalog, "Metrics Streaming Resources" isn't listed. Should this be [monitoring](https://docs.aws.amazon.com/controltower/latest/userguide/monitoring-overview.html)?

Reply:
No. This solution is not monitoring control tower. This solution is bootstrapping new accounts created by CT to ship metrics data using [CW metrics streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html)